Resources/Private/Language/locallang_db.xlf

```diff
@@ -453,6 +453,10 @@
 				<source>Page with policy details</source>
 			</trans-unit>
 
+			<trans-unit id="flexform.jobs.afterSubmitPage">
+				<source>Page to redirect to after successful submit ("Thank you"-Page)</source>
```

Comment:
The last should be `("Thank you" page)`.

Reply:
Done